### DISCUSSION TITLE
tpv33 Error: An HDF5 error occurred in PUML

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#592] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/104721358/176432805-1de9470e-1c3c-47cd-86d6-07c2cd9354e8.png)

**Comment by krenzland** [comment]
What's your question? The error message is pretty clear to me.

**Comment by Thomas-Ulrich** [comment, marked as answer]
you mesh file is not in hdf5 format... You need to run pumgen first.

**Reply by smallriver666** [reply]
> you mesh file is not in hdf5 format... You need to run pumgen first.

thank you.

**Reply by smallriver666** [reply]
> you mesh file is not in hdf5 format... You need to run pumgen first.
I run pumgen and then I encounter this problem
![image](https://user-images.githubusercontent.com/104721358/176625610-32833c47-139e-42cf-9f06-a93fc78f0cd6.png)

**Reply by krenzland** [reply]
Same reason as: https://github.com/SeisSol/SeisSol/discussions/558#discussioncomment-2853474

**Reply by smallriver666** [reply]
> Same reason as: [#558 (comment)](https://github.com/SeisSol/SeisSol/discussions/558#discussioncomment-2853474)

Ok,thank you.
